client.go

```diff
@@ -85,6 +86,7 @@ var (
 		"network_list": "/network-list/v1/network_lists",
 		"papi_v1":      "/papi/v1",
 		"reporting_v1": "/reporting-api/v1/reports",
+		"a2_v1":        "/adaptive-acceleration/v1/properties",
```

Comment:
Maybe we should use the same approach as with const type for env variables ?

Reply:
I can agree that having it in constants would be better. I will move only A2 one now. All others let's do via different pull request, as well as API removal from comment above